packages/restitution/src/HeaderRestitution.tsx

```diff
@@ -0,0 +1,65 @@
+import  * as React from 'react';
+import classNames from 'classnames';
+import {
+  PropsManager,
+  withClassDefault,
+  withClassModifier,
+  WithClassModifierOptions,
+} from '@axa-fr/react-toolkit-core';
+import { compose } from 'recompose';
+
+const DEFAULT_CLASSNAME = 'af-restitution__header';
+
+type HeaderRestitutionBaseProps = {
+  className?: string;
+  children?: React.ReactNode;
+} 
+
+const defaultProps: Partial<HeaderRestitutionBaseProps> = {
+  children: null,
+  className:DEFAULT_CLASSNAME
+};
+
+interface HeaderRestitutionProps extends WithClassModifierOptions, HeaderRestitutionBaseProps {}
+
+const HeaderRestitution: React.SFC<HeaderRestitutionProps> = ({ children, title, subtitle, titleLeft, className }: { children:React.ReactNode, title:React.ReactNode, subtitle:React.ReactNode, titleLeft:React.ReactNode, className:string }) => {
+  return (
+    <header className={className}>
+      <div className="af-restitution__header-left">
+        <div className="af-restitution__title">
+          {title && <span className="af-restitution__title-title">{title}</span>}
```

Comment:
"af-restitution__title-title" is ambigous, should be rename for "af-restitution__title-main" or other more explicit name.
This a default from https://axaguildev.github.io/react-toolkit/latest/design/#/restitution

Reply:
this update will make a breaking change. May be it would be a good idea to do it in another issues.